src/layout/RootLayout.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
```

Comment:
nothing new, just exported this so we can use QueryClient at the root layout which requires `use client` while we have the metadata in `layout.tsx`. This is so we can have queryclient for both dashboard and user pages